### PR TITLE
SAK-46973 Turnitin: Additional changes to the messages and recommenda…

### PR DESCRIPTION
**Pull request by mpellicer**
…tions.

This is related to changes requested on https://github.com/sakaiproject/sakai/pull/10289.

### CODE REVIEW

File: assignment/api/src/resources/assignment.properties [hunk 1]
```diff
@@ -255,7 +255,7 @@ gen.thesubswill   = The submissions will be sent to {0} to be reviewed for plagi
...
 gen.onlythefoll   = <li>Only the following file types will be accepted: {0}.</li>
-gen.onlythefoll.turnitin = <li>File extension should always be included. Text after first point will be considered as file extension.</li><li>Attachments should not exceed 100 MB in size (.txt attachments should not exceed 2 MB).</li><li>Attachments should occupy less than 800 pages.</li><li>The inclusion of vector images may result in problems when having the "originality report", we recommend .jpg format for images.</li>
+gen.onlythefoll.turnitin = <li>The uploaded file must have a file extension, which is the text following the dot.</li><li>Attachments must be less than 100 MB in size (.txt attachments should be less than 2 MB), have a minimum of 20 words, and contain less than 800 pages.</li><li>The inclusion of vector images may result in problems when having the "originality report", we recommend .jpg format for images.</li>
```
Comment (ottenhoff) [inline assignment/api/src/resources/assignment.properties:258]: I don't think you need to say "which is the text following the dot"

Comment (ottenhoff) [reply assignment/api/src/resources/assignment.properties:258]: "The inclusion of vector images may result in problems when having the "originality report", we recommend .jpg format for images."

This is so obscure. How many people know what vector images are?

Comment (mpellicer) [reply assignment/api/src/resources/assignment.properties:258]: @vacercon @josecebe I believe this comes from UPV, students from UPV uploading vector images and turnitin failing for this, am I right?

It's mentioned in the Turnitin support:
https://supportcenter.turnitin.com/s/article/M14-11-error

Comment (ottenhoff) [reply assignment/api/src/resources/assignment.properties:258]: Okay my opinion is that we should either be minimal and brief. Or we should list every single oddity/exception.

Comment (josecebe) [reply assignment/api/src/resources/assignment.properties:258]: I talked with @vacercon from UPV, and a proper solution would be to separate this vector images error to a list of possible technical errors.

**Review by baholladay — APPROVED** [review]
changes look fine and don't change the overall intended details/message... I would be worried about the translations (if there are any)

**Pull request merged**